yarn-project/acir-simulator/src/avm/opcodes/instruction.ts

```diff
@@ -2,14 +2,9 @@ import { AvmMachineState } from '../avm_machine_state.js';
 import { TypeTag } from '../avm_memory_types.js';
 import { AvmJournal } from '../journal/index.js';
 
-export const AVM_OPERAND_BYTE_LENGTH = 4; // Keep in sync with cpp code
-export const AVM_OPCODE_BYTE_LENGTH = 1; // Keep in sync with cpp code
-
-/**
- * Opcode base class
- */
 export abstract class Instruction {
```

Comment:
Can we replace that comment (which was wrong in the first place since it was the instruction base class) with a brief comment that accurately describes it?